src/main/java/net/sf/jabref/model/database/BibDatabase.java

```diff
@@ -562,4 +554,8 @@ public void setEpilog(String epilog) {
     public String getEpilog() {
         return epilog;
     }
+
+    public EventBus getEventBus() {
```

Comment:
I wouldn't expose the EventBus but provide a `register` (or `listenForChanges` or `subscribeToChanges` or whatever) method.